File: src/src/models/sourceComparer.ts
import { Package } from "./package";

export class SourceComparer {
    sourceA: string;
    sourceB: string;
    Packages: Package[];
    constructor(sourceA: string, sourceB: string, packages: Package[]) {
        this.sourceA = sourceA;
        this.sourceB = sourceB;
        this.Packages = packages;
    }

}

File: src/Gjallarhorn.Shared/SourceComparer.cs
using System.Collections.Generic;

namespace Gjallarhorn.Shared
{
    public class SourceComparer
    {
        public string SourceA{get;set;}
        public string SourceAAlias { get; set; }
        public string SourceB{get;set;}
        public string SourceBAlias { get; set; }
        public List<Package> Packages{get;set;}
    }
}